server/service/individualServices/AuthorizationApplication.js

```diff
@@ -268,8 +266,7 @@ async function ApplicationandReleaseNumber(applicationaName, ReleaseNumber, auth
                     let allowAcees = registeredApplication[allowedSccess]
```

Comment:
Correct the typo in the variable declaration from allowAcess to allowAccess. Also modify it in the used references.